mesonbuild/mparser.py

```diff
@@ -466,6 +470,12 @@ def __init__(self, linenode: BaseNode):
         self.ifs = []          # type: T.List[IfNode]
         self.elseblock = None  # type: T.Union[EmptyNode, CodeBlockNode]
 
+class TestCaseClauseNode(BaseNode):
+    def __init__(self, condition: BaseNode, block: CodeBlockNode):
+        super().__init__(condition.lineno, condition.colno, condition.filename)
+        self.condition = condition  # type: BaseNode
+        self.block = block          # type: CodeBlockNode
```

Comment:
```suggestion
        self.condition: BaseNode = condition
        self.block: CodeBlockNode = block
```

Reply:
Does not even need to repeate typing info here, it's already defined in the args of `__init__()`. Fixed.